modules/python/package/cv2/__init__.py

```diff
@@ -179,3 +179,13 @@ def load_first_config(fnames, required=True):
 
 
 bootstrap()
+
+# hotfix for pylint issue (https://github.com/opencv/opencv-python/issues/570)
+from . import getStructuringElement, MORPH_ELLIPSE
```

Comment:
We definitely should not have that.
Issue provides a minimal reproducer. Do you want to add here other 100500 symbols?

/cc @asmorkalov

Reply:
I've seen a problem only with these two. We didn't have this issue when we had used an import of anything in a binary, e.g.
```
from .cv2 import *
```
I guess there was the import of the same modules if the issue has been disappeared.

Reply:
Removed this line.